packages/dx-react-scheduler/docs/guides/appointments.md

```diff
@@ -1,6 +1,6 @@
 # React Scheduler - Appointments
 
-The Scheduler supports one-time and all-day appointments.
+The Scheduler supports one-time, recurring, zero-duration and all-day appointments.
```

Comment:
The Scheduler component can display a lot of appointment types to cover your work needs. There are one-time simple events, zero-duration, all-day and recurring appointments. @RomanTsukanov